src/metabase/pulse/render/body.clj

```diff
@@ -21,8 +21,23 @@
             [schema.core :as s])
   (:import [java.text DecimalFormat DecimalFormatSymbols]))
 
+(def ^:private card-error-rendered-info
+  "Default rendered-info map when there is an error running a card on the card run.
+  Is a delay due to the call to `trs`."
+  (delay {:attachments
+          nil
+
+          :content
+          [:div {:style (style/style
+                         (style/font-style)
+                         {:color       style/color-error
+                          :font-weight 700
+                          :padding     :16px})}
+           (trs "There was a problem with this question.")]}))
```

Comment:
I'm a bit fuzzy on the details of how the translation macros work, but I think the surrounding delay can be removed if you change this call to use `deferred-trs`

Reply:
Empirically you need to delay the whole shebang